src/features/dental.feature

```diff
@@ -1,6 +1,44 @@
 Feature: Test for Dental page
 
-  Scenario: Open Dental Insurance
+  Scenario: View Dental header elements 
     Given the user is on the Dental page
     When the Dental Insurance product page is opened
-    Then the main Dental title and description should be correct
\ No newline at end of file
+    Then the Dental title and description should be correct
+    And user can see Offerings link
+    And user can see Healthcare link
+    And find Offerings link is working
+    Then the Offerings page is opened
+    Then the user is on the Dental page
+    And find Healtcare link is working
+    Then the Healtcare page is opened
+
+  Scenario: User can go to Quizzard page from Dental page and from Quizzard page go back to the previuosly page
+    Given the user is on the Dental page
+    When the Dental Insurance product page is opened
+    Then find your product button for Dental? is working
+    Then the user is on the Quizzard page
+    When the Quizzard page is opened
+    Then validate back button label and click it
```

Comment:
Update name of this method to something like:
Then user clicks on back button, or Then user clicks Quizzard back button, user should be navigated back to Dental page